### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by rosieks** [input-output-hk/cardano-js-sdk#519] [General]
It would be nice to have documentation how to use all those libraries. I know there is already documentation for API it's rather useless. It doesn't any value that we already have with modern editors that support TypeScript.

**Comment by mkazlauskas** [comment]
`cardano-js-sdk` is still pre-release and under heavy development. Maintaining documentation would be a significant overhead at this point, since breaking are quite frequent. Feel free to ask any specific questions in [Discussions](https://github.com/input-output-hk/cardano-js-sdk/discussions).

**Comment by rhyslbw** [comment]
> It would be nice to have documentation how to use all those libraries.

We agree! Until the point of API stability, the `wallet` package [tests](https://github.com/input-output-hk/cardano-js-sdk/tree/9ca2361c56ea42bbcd36c73b4ef98a31264d3907/packages/wallet/test) demonstrates most of the lower-level packages, and serves as a good starting point.
